yeonjae/programmers-1/가운데글자가져오기.js

```diff
@@ -0,0 +1,11 @@
+function solution(s) {
+  const AverageIdx = Math.floor(s.length / 2);
+      
+  if (s.length % 2 === 0) {
+    return s[AverageIdx - 1] + s[AverageIdx]
+  }
+  else {
+    return s[AverageIdx]
+  }    
+}
+  
```

Comment:
```suggestion
function solution(s) {
  const AverageIdx = Math.floor(s.length / 2);
      
  if (s.length % 2 === 0) {
    return s[AverageIdx - 1] + s[AverageIdx];
  }
  return s[AverageIdx];
}
```
<br />

- 오우 저는 slice로 짜를 생각을했는데 그냥 단순히 문자열을 `+`연산자로 붙이면 되는군요! 잘봤습니다 👍
- 피드백을 약간 드리자면 문장의 끝에`세미콜론(;)`을 붙여주는 습관 들여주세요! 
- 그리고 `if... else`문에서 첫 번째 조건식에서 return문이 있으면 조건에 해당하면 바로 return을 해서 함수를 종료하고, 아니면 다음 코드가 실행되는데 else문을 만들어서 거기서 return을 하는 것과 else문 없이 바로 return을 하는 것이 동일하기 때문에 else문은 제거해주셔도 됩니다.

Reply:
바로 Math.floor(s.length / 2) 를 변수에 담아 사용하는 것이 더 깔끔하네요!!!! 도움이 많이 되었습니당👍🏻👍🏻